css/index.css

```diff
@@ -0,0 +1,79 @@
+@media screen and (min-width: 480px) {
+
+
+	html {
+		display: flex;
+		flex-flow: row wrap;
+		justify-content: space-around;
+	
+		background-color: black;
+		font-family: "Times New Roman", Times, serif;
+		color: green;
+		padding: 1px;
+		margin: 1px;
+	}
+}
+
+header {
+    background-color: gray;
+    color: white;
+    padding: 1px;
+}
+
+body {
+    background-color: #f8f8ff;
+}
+
+section {
+
+}
+
+section ul {
+
+}
+
+a {
+
+}
+
+a:hover {
+ 
+}
+
+h2 {
+	color: orange;
+	font-size: 16pt;
+}
+
+
+button {
+    padding: 10px;
+    margin: 1px;
+    color: black;
+    border-radius: 10px;
+    border:1px solid black;
+}
```

Comment:
One of the tasks included changing the background color of the button. (Remember that `color` refers to the font.)